edgelet/iotedge/src/support_bundle.rs

```diff
@@ -219,24 +219,24 @@ where
             NaiveDateTime::from_timestamp(state.log_options.since().into(), 0),
             Utc,
         );
-        let since = since_time.format("%F %T").to_string();
```

Comment:
Is there a reason this fix was needed for `Get-WinEvent` but not for `Get-EventLog` used by `write_docker_log_to_file` ?

Reply:
It might have been needed, I just didn't notice b/c the docker log has never been more than 1 or 2 lines from when it was first started. I'll go ahead and make a change for that too.